content/post/085-announcing-rust-ipfs.md

```diff
@@ -0,0 +1,66 @@
+
+## Why Rust IPFS? Why Now?
+
+Rust, the programming language, has enjoyed a recent spike in popularity. This is due both to its
+inclusive community, and also being a safe systems language with performance comparable to C and
+C++. An IPFS implementation written in Rust only makes sense. Additionally, the community has been
+[asking for a while now](https://github.com/ipfs/notes/issues/363), for a number of use cases:
+
+1. Usage in resource-constrained systems such as Industrial Internet-of-Things (IIoT) controllers
+2. Portability of certain subsets of the IPFS stack into WebAssembly
+3. Usage of IPFS functionality and APIs via native Rust function calls via inclusion as a Rust crate.
+
+Given that, it's no surprise that Rust-IPFS was also one of the [IPFS 2020 Theme Proposals](https://github.com/ipfs/roadmap/issues/54) suggested in our community-driven [IPFS 2020 Roadmapping Process](https://github.com/ipfs/roadmap/blob/master/2020-IPFS-Project-Planning.md) - and that the community quickly rose to the challenge. Equilibrium, an active participant in many dweb infrastructure projects, stepped up to take on core implementation and stewardship of this new language implementation, with support through the [new IPFS DevGrants program](https://github.com/ipfs/devgrants) to provide coordination and maintainership for passionate developers in the wider Rust community. 
+
+With both community demand and active participation, the time was ripe for the Rust-IPFS effort to be revitalized and reborn!
+
+
+## Leveraging Community Efforts
+
+The IPFS community is a talented group of people with an impressive array of high quality work in the Rust space already. There's the aforementioned
+[rust-libp2p](https://github.com/libp2p/rust-libp2p), which is already used by the likes of
+Polkadot, Substrate, and Lighthouse. We have ferrismtg's
+[rust-ipfs-api](https://github.com/ferristseng/rust-ipfs-api) which is aleady doing wonders in
+providing a bridge from the Rust world. Finally, we have David Craven's
+[rust-ipfs](https://github.com/ipfs-rust/rust-ipfs), which is the foundational work behind this latest endeavor.
+
+Instead of forking and/or starting fresh, this project aims to support and build on the impressive amount of progress that's already been made, helping carry the torch across the finish line while _incorporating_ and _including_ the community of passionate Rust contributors in the process.
+the finish line, _incorporating_ and _including_ the community into its efforts and not forking
+and/or starting fresh.
+
+## What can be expected, and when?
+
+As you can see in the [Rust-IPFS DevGrant Roadmap](https://github.com/ipfs/devgrants/tree/master/open-grants/ipfs-rust) implementation work
+takes place mostly over early Q2 2020 and covers the use case of **IPLD applications**,
+targeting the low-level blockstore, libp2p integration, and IPLD-related functionality
+such as `ipfs dag put` and `ipfs dag get`. 
```

Comment:
Suggestion: add in an image of the three phases of the roadmap at a high level?

Reply:
I think it's a great idea, though I'm not personally a graphic designer...

Reply:
```suggestion
such as `ipfs dag put` and `ipfs dag get`. 

![Rust IPFS Timeline](https://gateway.ipfs.io/ipfs/QmU7sssvo52Rrwj7MWZNpeHnFjjdG271Dx5zfGkZSbgVnN)
```